run-ganache.sh

```diff
@@ -3,6 +3,6 @@
 docker stop namebazaar-ganache || true
 docker rm namebazaar-ganache || true
 
-docker run --name=namebazaar-ganache -d \
+docker run --name=namebazaar-ganache \
```

Comment:
Nice. Although I hardly believe anything meaningful will come out of those logs :/